src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx

```diff
@@ -47,7 +47,7 @@ describe('facilities <FacilityPatientSatisfactionScoresWidget>', () => {
       const primaryUrgentScore = tree.find(
         '#facility-patient-satisfaction-scores-primary-urgent-score',
       );
-      expect(primaryUrgentScore.text()).to.contain('N/A');
+      expect(primaryUrgentScore).to.not.have.length;
```

Comment:
You could do `expect(primaryUrgentScore.isEmpty()).to.be.true;` I think that's maybe clearer?

Reply:
i will do these in my re-factor for 2271